haas/ext/obm/ipmi.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013-2014 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""
+A obm driver for implementing lego robot based controls for 
+arm based servers on in a DC at North Pole :-)
```

Comment:
yup. I had corrupted my ipmi driver while trying to fix it. So recreated it using the lego.py template. Missed out on changing the docstring. 

Corrected it.